package.json

```diff
@@ -112,7 +112,7 @@
     "less-loader": "^11",
     "lint-staged": "^13.0.2",
     "lintspaces-cli": "^0.8.0",
-    "marko": "^5",
+    "marko": "^5.23.0",
```

Comment:
Leave this as version 5, update only package.json. 

Is this upgrade needed?
Also do we need a peer depdency of marko or not necessarily?